explorer/graphql/app/schema.graphql

```diff
@@ -118,6 +124,7 @@ type Query {
   xblock(sourceChainID: BigInt!, height: BigInt!): XBlock
   xblockrange(amount: BigInt!, offset: BigInt!): [XBlock]!
   xblockcount: BigInt
+  xmsgrange(amount: BigInt!, offset: BigInt!): [XMsg]!
```

Comment:
How many messages do we potentially return here? Can this be a huge number that absolutely needs pagination?